### DISCUSSION TITLE
MLX-my-repo - a no-code way to create MLX quants (Q4/ Q8)

### DISCUSSION THREAD

**Discussion by Vaibhavs10** [ml-explore/mlx-lm#1] [Show and tell]
Hi all,

I'm VB, I am a GPU Poor in residence at Hugging Face and a big fan of MLX. Recently we put together a Space that allows you to create MLX quants directly on the Hugging Face Hub without the need to download anything or setup an environment.

The space is available here: https://huggingface.co/spaces/mlx-community/mlx-my-repo 🔥

![image](https://github.com/user-attachments/assets/2c375ce2-8cee-44ee-a035-6a31abd5c48c)

Note: We don't keep a log of who creates what quants, so it's privacy first and you can even create private model checkpoints too!

This is an initial version with a bit of rough edges, but keen to hear your feedback on this. 🤗

**Comment by fblissjr** [comment]
Nice - thanks @Vaibhavs10. Helps us upload-speed-poors as well.

**Comment by DePasqualeOrg** [comment]
@Vaibhavs10, your Space does not use the established naming convention for MLX repos, and this is resulting in repos being uploaded to MLX Community which use a divergent naming convention. Please fix your tool to use the established naming convention like this:

Phi-4-mini-instruct-4bit

**Comment by reneleonhardt** [comment]
Thank you @Vaibhavs10!
Where is the GitHub repo to allow the community to contribute?

**Reply by LumiWasTaken** [reply]
> Thank you @Vaibhavs10! Where is the GitHub repo to allow the community to contribute?

https://huggingface.co/spaces/mlx-community/mlx-my-repo/tree/main
